LEGO1/lego/legoomni/src/race/legoracers.cpp

```diff
@@ -332,11 +376,80 @@ void LegoRaceCar::VTable0x70(float p_float)
 	}
 }
 
-// STUB: LEGO1 0x100133c0
+// FUNCTION: LEGO1 0x100133c0
+// FUNCTION: BETA10 0x100cbb84
 MxResult LegoRaceCar::VTable0x94(LegoPathActor* p_actor, MxBool p_bool)
 {
-	// TODO
-	return 0;
+	if (!p_actor->GetUserNavFlag()) {
+		if (p_actor->GetState()) {
+			return FAILURE;
+		}
+
+		if (p_bool) {
+			LegoROI* roi = p_actor->GetROI(); // name verified by BETA10 0x100cbbf5
+			assert(roi);
+			MxMatrix matr;
+			matr = roi->GetLocal2World();
+
+			Vector3(matr[3]).Add(g_vector020);
+			roi->FUN_100a58f0(matr);
+
+			p_actor->SetState(2);
+		}
+
+		if (m_userNavFlag) {
+			MxBool actorIsStuds = strcmpi(p_actor->GetROI()->GetName(), "studs") == 0;
+			MxBool actorIsRhoda = strcmpi(p_actor->GetROI()->GetName(), "rhoda") == 0;
+			MxTimer* timer = Timer();
```

Comment:
This shouldn't be a variable. `Timer()->GetTime()` is how it's usually written

Reply:
The version with the variable matches slightly better at `0x10013434`. I'll change it nevertheless.